front/src/app/api.ts

```diff
@@ -1,7 +1,7 @@
 import createAPI from "@/lib/api";
 
 const api = createAPI({
```

Comment:
- [ ] Can you please add something like the following:

```typescript
const baseURL =
  process.env.NODE_ENV === "development"
    ? process.env.NEXT_PUBLIC_BACKEND_HOST || ""
    : "";
```

This way the environment variable is used in development (when the backend and frontend are separated) but not in production (when the backend serves the frontend in the same domain).